ARM/Linux build warning fix related with unw

If we try to cross build ARM/Linux binaries of CoreCLR, we get the following clang warning, which is set to stop building.

```
[  4%] In file included from /source/tizen_3.0/dotnet/coreclr/src/pal/src/exception/seh.cpp:321:
/source/tizen_3.0/dotnet/coreclr/src/pal/src/exception/seh-unwind.cpp:248:10: error: use of SP or PC in the list is deprecated [-Werror,-Winline-asm]
    st = unw_getcontext(&unwContext);
         ^
/source/tizen_3.0/dotnet/coreclr_upstreaming/cross/rootfs/arm//usr/include/libunwind-common.h:108:29: note: expanded from macro 'unw_getcontext'
#define unw_getcontext(uc)              unw_tdep_getcontext(uc)
                                        ^
/source/tizen_3.0/dotnet/coreclr_upstreaming/cross/rootfs/arm//usr/include/libunwind-arm.h:279:6: note: expanded from macro 'unw_tdep_getcontext'
    "stmia %[base], {r0-r15}\n"                                         \
     ^
<inline asm>:5:1: note: instantiated into assembly here
stmia r0, {r0-r15}
^
```

--- a/src/pal/src/CMakeLists.txt
+++ b/src/pal/src/CMakeLists.txt
@@ -88,6 +88,9 @@ elseif(PAL_CMAKE_PLATFORM_ARCH_ARM64)
   )
 endif()
 
+if(CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM)
+set_source_files_properties(exception/seh.cpp PROPERTIES COMPILE_FLAGS -Wno-error=inline-asm)
+endif(CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM)
 set(SOURCES
   cruntime/file.cpp
   cruntime/filecrt.cpp

